velox/expression/tests/ExpressionFuzzer.h

```diff
@@ -148,6 +152,11 @@ class ExpressionFuzzer {
   /// particular iteration.
   std::vector<TypePtr> inputRowTypes_;
   std::vector<std::string> inputRowNames_;
+
+  /// Maps a 'Type' serialized as a string to the column names that have already
```

Comment:
Is there a better way to use a Type object as a key? I could try to write a custom hasher using Type::hashKind() and Type::operator==().
what do you think?

Reply:
Serializing the type to string as the key looks good to me. TypeKind may not capture all information about custom types (that inherit native types and hence have the same type kind).